coalib/processes/Processing.py

```diff
@@ -452,12 +452,16 @@ def yield_ignore_ranges(file_dict):
                                    line_number,
                                    len(file[line_number-1])))
                 elif "ignore " in line:
+                    # When using a plain "ignore" we just want to exclude
+                    # analysis in the following line. And if there's no
+                    # following line, just ignore the ignore-comment itself.
+                    end_line = min(line_number + 1, len(file))
                     yield (get_ignore_scope(line, "ignore "),
-                           SourceRange.from_values(filename,
-                                                   line_number,
-                                                   1,
-                                                   line_number+1,
-                                                   len(file[line_number])))
+                           SourceRange.from_values(
+                               filename,
+                               line_number, 1,
+                               end_line, len(file[end_line - 1])))
```

Comment:
if you just give it an own line I don't see why we shouldn't inline the expression her